docs/labs/flockercontainers.rst

```diff
@@ -0,0 +1,96 @@
+.. _flocker-containers:
+
+=============================
+Running Flocker in Containers
+=============================
+
+The :ref:`Flocker Installer <labs-installer>` allows you install Flocker in a variety of environments.
```

Comment:
This isn'y necessarily tied to the labs installer. You can install flocker this way by hand, on CoreOS and ECS etc... thoughts?